README.md

```diff
@@ -84,7 +84,7 @@ TEAMMATES team wishes to thank the following invaluable contributions.
 * [**School of Computing, National University of Singapore (NUS)**](http://www.comp.nus.edu.sg), for providing us with the infrastructure support to run the project.
 * [**Centre for Development of Teaching and Learning (CDTL)**](http://www.cdtl.nus.edu.sg/) of NUS, for supporting us with several *Teaching Enhancement Grants* over the years.
 * **Learning Innovation Fund-Technology (LIF-T)** initiative of NUS, for funding us for the 2015-2018 period.
-* **Google Summer of Code** Program, for including TEAMMATES as a mentor organization in *GSoC2014* and *GSoC2015* editions.
+* **Google Summer of Code** Program, for including TEAMMATES as a mentor organization in *GSoC2014*, *GSoC2015* and *GSoC2016* editions.
```

Comment:
Can you also mention about Facebook Open Academy (as how it is done in the about page) here? Something like:

* **Facebook Open Academy** Program, for including TEAMMATES as a mentor organization in *FBOA 2016*.